src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -33,23 +33,23 @@
     public static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
 
     public static final Pattern ADD_ITEM_DATA_ARGS_FORMAT =
-            Pattern.compile("n/(?<itemName>[^/]+)"
-                    + " shlv/(?<shelfName>[^/]+)"
-                    + " p/(?<purchaseCost>([0-9]+([.][0-9]{1,2})?))"
-                    //only accepts numbers or decimals in 1 or 2 d.p.
-                    + " s/(?<sellingPrice>([0-9]+([.][0-9]{1,2})?))"
-                    //only accepts numbers or decimals in 1 or 2 d.p.
-                    + " q/(?<quantity>[0-9]+)" // only accepts integers, no decimals
-                    + "( r/(?<remarks>[^/]+))?$"); // optional argument
```

Comment:
https://se-education.org/guides/conventions/java/basic.html#layout

Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line.

Reply:
noted